Yoast/wpseo-news
# Introduces a googlebot-news metatag presenter

## Context


* See #623

## Summary

This PR can be summarized in the following changelog entry:

* Moves the rendered metatag for `Googlebot-News` inside the Yoast marks. 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Ensure you have posts as news and noindex a post from the Googlebot news.
* Checkout trunk and see the tag is rendered at the wrong place. Outside the yoast debug marks.
* Checkout this branch and see it is rendered inside at the end of all our metatags

Fixes #623 
Needs https://github.com/Yoast/wordpress-seo/pull/14696
